api/peripheral_api/routes/Printer.js

```diff
@@ -88,4 +91,32 @@ router.post('/sendPrintRequest', async (req, res) => {
   });
 });
 
+router.post('/pushDiscordPDF', async (req, res) => {
+  const { apiKey, fileURL } = req.body;
+
+  if(apiKey !== DISCORD_PRINTING_KEY){
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  const sqs = new AWS.SQS({apiVersion: '2012-11-05'});
+
+  const accountId = ACCOUNT_ID;
+  const queueName = PAPER_PRINTING_QUEUE_NAME;
+
+  const sqsParams = {
+    MessageBody: JSON.stringify({
+      fileURL
+    }),
```

Comment:
can put it all on one line
```suggestion
    MessageBody: JSON.stringify({ fileURL }),
```